recipes/qtconsole/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: qtconsole
+  version: {{ version }}
+
+source:
+  fn: qtconsole-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/q/qtconsole/qtconsole-{{ version }}.tar.gz
+  md5: c08ebebc7a60629ebadf685361ca0798
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - ipykernel >=4.1
+    - jupyter_client >=4.1
+    - jupyter_core
+    - pygments
+    - pyqt
+    - traitlets
+
+  run:
+    - python
+    - ipykernel >=4.1
+    - jupyter_client >=4.1
+    - jupyter_core
+    - pygments
+    - pyqt
+    - traitlets
+
+test:
+  commands:
+    - jupyter qtconsole --help                                              # [not linux]
+    - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'jupyter qtconsole --help'  # [linux]
```

Comment:
Kind of gross, but this demonstrates that we can connect the X Windows Server and can run something simple.